src/code/dialogs/confirmDialog.js

```diff
@@ -46,7 +46,11 @@ const ConfirmDialog = WDialog.extend({
 
   removeHooks: function () {
     WDialog.prototype.removeHooks.call(this);
-    window.map.fire("wasabeeUIUpdate", { reason: "confirmDialog" }, false);
```

Comment:
should this be handled by the `callback` instead ?

Reply:
yes, did a sweep to make this change